actions/class.Runner.php

```diff
@@ -128,11 +130,16 @@ protected function getServiceContext()
     {
         if (!$this->serviceContext) {
             $testExecution = $this->getSessionId();
+            /** @var DeliveryExecution $execution */
             $execution = $this->getServiceLocator()->get(DeliveryExecutionService::SERVICE_ID)->getDeliveryExecution($testExecution);
+            $userIdentifier = common_session_SessionManager::getSession()->getUser()->getIdentifier();
```

Comment:
Using this static call is really the only way? Could not we get from serviceLocator instead / also is there any test for this?